### ISSUE TITLE
[BUG] Since update to 1.12.2 Barcode splitting not work anymore

### ISSUE THREAD

**Issue by Mannshoch** [paperless-ngx/paperless-ngx#2719]
### Description

Until 1.12.2 I used a self generated barcode for scanning. Since 1.12.2 it works not anymore. Now I tryed to use a native document linked in the documentation. Here I use the PATCH2 A4 Page
```
PAPERLESS_CONSUMER_BARCODE_STRING=PATCH2
PAPERLESS_CONSUMER_ENABLE_BARCODES=true
PAPERLESS_CONSUMER_BARCODE_TIFF_SUPPORT=true
```
but nothing happen.

Setup: [docker-compose.sqlite-tika.yml](https://github.com/paperless-ngx/paperless-ngx/blob/main/docker/compose/docker-compose.sqlite-tika.yml)

### Steps to reproduce

1. Scan multible Pages seperated by a Barcode
2. Only one Document appear

### Webserver logs

```bash
there is no Barcode Entry.
```

### Paperless-ngx version

1.12.2

### Host OS

Linux NAS 4.4.180+ #42962 SMP Tue Sep 20 22:35:56 CST 2022 x86_64 GNU/Linux synology_geminilake_220+

### Installation method

Other (please describe above)

### Configuration changes

replaced Volumes, replaced docker-compose.env with .env

**Comment by stumpylog** [comment]
Without a sample file, there's no testing we can do.  Likely, the barcode isn't detected and there's very little we can do about that.

**Comment by Mannshoch** [comment]
[PATCH2.pdf](https://github.com/paperless-ngx/paperless-ngx/files/10785455/PATCH2.pdf)
Attached the split page my scanner scanned and paperless did not recognize.

**Comment by stumpylog** [comment]
A document with that page in the middle splits as expected for me.

**Comment by Mannshoch** [comment]
[2023-02-20 2023-02-20.16-16.pdf](https://github.com/paperless-ngx/paperless-ngx/files/10785986/2023-02-20.2023-02-20.16-16.pdf)

**Comment by stumpylog** [comment]
That file also splits correctly in 1.12, 1.13 and dev.

A common cause for this is having changed environment variables, but only having restarted, not recreated the container.

**Comment by Mannshoch** [comment]
Ha!!
Yes, that was the missing hint. The Problem was not a missed recreated container. It was the .env file.
I recently played around with userrights. That Day, nothing I did worked. Until I detected, that all tutorials I fount on Internet worked with a .env. So I renamed the docker-compose.env to .env. My User right problems disappeared. But now I found that I have to add the .env also into the .yml file.

It seems I still do not fully understand docker. I was not able to found a hint what's the difference between .env and a named .env
